src/Services/Analytics.ts

```diff
@@ -91,6 +91,20 @@ class AnalyticsService {
       amount,
     });
   }
+
+  transferUndefinedTxHashFromFallback({
+    address,
+    recipient,
+    nonce,
+    amount,
+  }: TransferEventData) {
+    this.ga.trackEvent("transfer_undefined_tx_hash_from_fallback", {
```

Comment:
How can we detect the direction?

Reply:
We can detect direction by address. Also, we can add a specific variable to all analytics methods.

Reply:
Ok.
Please, create a task for the enhancement.